account-notifications.html

```diff
@@ -1923,7 +1923,7 @@ <h3 class="footer__heading">Airbnb</h3>
                           </section>
                         </div>
                         <div class="footer__copyright">
-                          <span class="a8jt5op dir dir-ltr"
+                          <span class="text_highlight dir dir-ltr"
```

Comment:
```suggestion
                          <span class="text-highlight dir dir-ltr"
```

use dashes as separator